resources/aws-rds/README.md

```diff
@@ -16,6 +16,8 @@ Add `shuttle-aws-rds` to the dependencies for your service. Every engine is behi
 | MySql    | mysql        | shuttle_aws_rds::MySql    |
 | MariaDB  | mariadb      | shuttle_aws_rds::MariaDB  |
 
+Note: You can also add `-rustls` after the feature flag, `postgres-rustls` etc
```

Comment:
Perhaps we should expand on this a bit and mention that it will use sqlx with native-tls by default, you can add rustls if you'd rather use that.